js/privatebin.js

```diff
@@ -655,9 +655,12 @@ $(function() {
                         this.clearText.html(
                             converter.makeHtml(text)
                         );
+                        // add table class from bootstrap css
+                        this.clearText.find('table').addClass('table');
+
                         this.clearText.removeClass('hidden');
+                        this.prettyMessage.addClass('hidden');
                     }
-                    this.prettyMessage.addClass('hidden');
```

Comment:
The prettyMessage should be hidden in any case, not just when showdown was found. Otherwise the decoded message is displayed twice (once in the clearText element and once in the prettyPrint one).